sand/extensions/search/default_search.py

```diff
@@ -0,0 +1,73 @@
+from typing import List
+import re
+from sand.extension_interface.search import IEntitySearch, IOntologySearch
+from sm.misc.funcs import import_attr
+from sand.models.search import SearchResult
+from sand.config import SETTINGS
+
+
+class DefaultSearch(IEntitySearch, IOntologySearch):
+
+    def local_search(self, default_entities: dict, search_text: str) -> List:
+        """ performs local partial text search across default entities"""
+        query_tokens = re.findall("[a-zA-Z]+", search_text)
+        search_results = []
+        for key, entity in default_entities.items():
+            if all(list(map(lambda x: x.lower() in entity.label.lower(), query_tokens))):
+                search_results.append(entity)
+        return search_results
+
+    def find_class_by_name(self, search_text: str) -> List[SearchResult]:
+        """Uses local default classes to search for entities using their name/text."""
+        constructor = SETTINGS['ont_classes']['default']
+        default_classes = import_attr(constructor)
```

Comment:
why import the default classes here? it's inefficient. put it in the __init__

Reply:
added them as instance variables.